grails-app/services/org/pih/warehouse/data/DataService.groovy

```diff
@@ -803,9 +803,21 @@ class DataService {
 
     def transformObject(Object object, Map includeFields) {
         Map properties = [:]
+        def value = null
```

Comment:
I feel like this needs to be inside the `each` so we clear the value on each iteration and don't do something that accidentally causes `value` to contain data from a previous iteration.